deployments/quick-start/kubernetes-vault-rbac.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-vault
+  namespace: default
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: kube-vault
+rules:
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs: ["get", "list"]
```

Comment:
Kubernetes-Vault watches for new pods using https://kubernetes.io/docs/api-reference/v1.6/#watch-list-all-namespaces-66. Is it necessary to have the `get` and `list` verbs, or can we just use `watch`?

Also, Kubernetes-Vault needs to be able to `list` the `endpoints` to discover other instances of Kubernets-Vault using https://kubernetes.io/docs/api-reference/v1.7/#list-132